pages/restaurant/[restaurantId].js

```diff
@@ -0,0 +1,41 @@
+import { Box } from '@mui/material';
+import {useRouter} from "next/router";
+import RestaurantInfoHeader from "../../components/restaurantInfo/components/restaurantInfoHeader";
+import RestaurantInfoContent from "../../components/restaurantInfo/components/restaurantInfoContent";
+import Contact from "../../components/restaurantInfo/components/contact";
+import OpeningHours from "../../components/restaurantInfo/components/openingHours";
+import {useEffect, useState} from "react";
+import getRestaurantById from "../api/restaurantService";
+
+const RestaurantInfoPage = () => {
+    const [restaurantData, setRestaurantData] = useState(null);
+    const router = useRouter();
+    const { restaurantId } = router.query;
+
+    useEffect(() => {
+        const fetchData = async () => {
+            try {
+                console.log("Restaurant ID:", restaurantId);
+                const data = await getRestaurantById(restaurantId);
```

Comment:
here when trying to fetch data from backend, always meet error in Chrome:
```
Access to XMLHttpRequest at 'http://localhost:8080/v1/public/restaurants/1' from origin 'http://localhost:3000' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource.
axios.js:23
```

Seems need to change the config in backend of CROS, but don't know how to solve this.